Formula/ignition-tools1.rb

```diff
@@ -0,0 +1,41 @@
+class IgnitionTools < Formula
```

Comment:
@scpeters may be better to speak about this, but I think we shouldn't touch the v1 formula. i.e. just leave `ignition-tools.rb` for that.

Reply:
Hmm, let me point tools1 to the `ignition-tools1.rb` and tools2 to the other one?

Reply:
https://github.com/osrf/homebrew-simulation/pull/1888/commits/ef7c8336687306c4830a744772f548c1a90c4757

https://github.com/osrf/homebrew-simulation/pull/1888/commits/c953959d2121a9b38e8618cc4a3c83bc97b8a864

Reply:
this file should be deleted

Reply:
https://github.com/osrf/homebrew-simulation/pull/1888/commits/cdfd567ea7eac97ff4dd046005076b185e05408b